### PR TITLE
DEV: Run angle-bracket codemod

### PR DESCRIPTION
**Pull request by wagenet**
Includes manual correction of `{{action}}` to `{{this.action}}` and renaming a local template variable from `log` to `logItem`.

I also ran a second pass with the following config:

```json
{
  "components": [
    "ace-editor",
    "admin-backups-logs",
    "admin-editable-field",
    "admin-form-row",
    "admin-graph",
    "admin-nav",
    "admin-penalty-history",
    "admin-report",
    "admin-report-table-cell",
    "admin-report-table-header",
    "admin-report-table-row",
    "admin-theme-editor",
    "admin-user-field-item",
    "admin-web-hook-event",
    "admin-web-hook-event-chooser",
    "admin-web-hook-status",
    "admin-wrapper",
    "basic-topic-list",
    "bread-crumbs",
    "categories-admin-dropdown",
    "category-notifications-button",
    "color-example",
    "combo-box",
    "composer-presence-display",
    "cook-text",
    "create-topic-button",
    "d-button",
    "d-modal",
    "d-modal-body",
    "d-icon",
    "d-section",
    "dashboard-new-features",
    "dashboard-new-feature-item",
    "dashboard-problems",
    "date-input",
    "date-picker",
    "date-time-input",
    "discourse-local-dates-create-form",
    "email-styles-editor",
    "embeddable-host",
    "embedding-setting",
    "emoji-value-list",
    "empty-state",
    "flag-user",
    "footer-message",
    "group-chooser",
    "highlighted-code",
    "homepage-preview",
    "inline-edit-checkbox",
    "input-tip",
    "install-theme-item",
    "invite-list-user",
    "list-setting",
    "mobile-nav",
    "mount-widget",
    "multi-select",
    "navigation-bar",
    "password-field",
    "penalty-post-action",
    "permalink-form",
    "poll-breakdown-chart",
    "poll-breakdown-option",
    "radio-button",
    "screened-ip-address-form",
    "secret-value-list",
    "setting-validation-message",
    "signup-cta",
    "silence-details",
    "simple-list",
    "site-customization-change-details",
    "site-customization-change-field",
    "site-setting",
    "site-text-summary",
    "staff-actions",
    "styleguide-example",
    "styleguide-icons",
    "styleguide-link",
    "styleguide-section",
    "suggested-topics",
    "suspension-details",
    "text-field",
    "theme-setting-relatives-selector",
    "theme-setting-editor",
    "theme-translation",
    "themes-list",
    "themes-list-item",
    "timezone-input",
    "topic-footer-buttons",
    "topic-list",
    "topic-presence-display",
    "user-link",
    "user-notifications-dropdown",
    "value-list",
    "version-checks",
    "watched-word-form",
    "watched-word-uploader",
    "wizard-canvas",
    "wizard-field",
    "wizard-step",
    "wizard-step-form"
  ]
}
```

Note that this does not yet update the tests.

### CODE REVIEW

**Comment by pmusaraj** [comment]
This is great, I pulled the branch locally and did not see any issues. Given the size, we should still be careful when merging, though. Maybe we can run a merge branch for a day or two on a few of our own instances @janzenisaac @CvX?

File: app/assets/javascripts/admin/addon/templates/backups-index.hbs [hunk 1]
```diff
@@ -1,26 +1,14 @@
...
 
   {{#if site.isReadOnly}}
-    {{d-button
-      class="btn-default"
-      icon="far-eye"
-      action=(action "toggleReadOnlyMode")
-      disabled=status.isOperationRunning
-      title="admin.backups.read_only.disable.title"
-      label="admin.backups.read_only.disable.label"}}
+    <DButton @class="btn-default" @icon="far-eye" @action={{action "toggleReadOnlyMode"}} @disabled={{status.isOperationRunning}} @title="admin.backups.read_only.disable.title" @label="admin.backups.read_only.disable.label" />
```
Comment (pmusaraj) [inline app/assets/javascripts/admin/addon/templates/backups-index.hbs:9]: Cosmetic issue, but is there a chance we can avoid having these super-long one-liners? Or even keep the same indentation as before?

Comment (CvX) [reply app/assets/javascripts/admin/addon/templates/backups-index.hbs:9]: Yeah, we'll have to live with this for a bit. It's gonna be resolved by CI/prettier changes in a couple of weeks (hopefully :) )

**Review by CvX — APPROVED** [review]
Looks good! I'm going to merge it in tomorrow morning (UTC+2 time) if nobody protests it before that 😃 cc: dunno… @jjaffeux, @tgxworld? 😛

**Comment by CvX** [comment]
Ok, merged it in 16ea7ab248f1bc1e2e0acfd6297ddf16a4a43ec1/#17233

@wagenet Hopefully GH will still recognize you as a contributor now, so we don't have to manually approve CI runs 😛

Btw. it might be useful to enable "Allow edits from maintainers" on future PRs. e.g. in case of merge conflicts like here 😃

**Pull request closed without merging**